WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadHandler.java

```diff
@@ -280,16 +280,6 @@ private void prepareUploadAnalytics(String postContent) {
                 sCurrentUploadingPostAnalyticsProperties = new HashMap<>();
                 sCurrentUploadingPostAnalyticsProperties
                         .put("word_count", AnalyticsUtils.getWordCount(mPost.getContent()));
-                sCurrentUploadingPostAnalyticsProperties.put("editor_source",
```

Comment:
Similar to a comment earlier, do we intentionally remove the `editor_source` property?